Makefile

```diff
@@ -138,7 +144,10 @@ $(OUT_DIR)/$(DOCKER_BUILDER): $(GO_SRC) $(BPF_SRC) $(MAKEFILE_LIST) Dockerfile |
 
 # docker_builder_make runs a make command in the tracee-builder container
 define docker_builder_make
-	$(CMD_DOCKER) run --rm -v $(abspath $(DOCKER_BUILDER_KERN_SRC_MNT)):$(DOCKER_BUILDER_KERN_SRC_MNT) -v $(abspath .):/tracee --entrypoint make $(DOCKER_BUILDER) KERN_SRC=$(DOCKER_BUILDER_KERN_SRC) $(1)
+	$(CMD_DOCKER) run --rm \
+	-v $(abspath $(DOCKER_BUILDER_KERN_SRC_MNT)):$(DOCKER_BUILDER_KERN_SRC_MNT) \
+	-v $(abspath .):/tracee \
+	--entrypoint make $(DOCKER_BUILDER) KERN_BLD_PATH=$(DOCKER_BUILDER_KERN_BLD) KERN_SRC_PATH=$(DOCKER_BUILDER_KERN_SRC) $(1)
```

Comment:
when adding line breaks, there are cases where this might not work properly, like when this function is called form a substitute. we shouldn't have any of those so it's not a big deal

Reply:
So what do you suggest? If it's not a big deal should we keep it with line breaks? (otherwise line is really long)

Reply:
I have has this issue with the `release_docker_image` function, where adding line-breaks broke the caller, because of how it was being called. I guess either way is fine.